packages/dashboard/src/components/side-panel/index.css

```diff
@@ -0,0 +1,3 @@
+.iot-side-panel {
+  height: 100%;
```

Comment:
Make sure we try this on different screen sizes, especially tablet/mobile if we're trying to support it. Percentage heights get funky sometimes with mobile browsers.